COMRS-191  import necessary concepts so that Indicator 2 report can b…

IipperMangesh opened the pull request:
Create CSV  and maintain hierarchy, 
FICHE DU PATIENT AVEC VIH - ADULTE ---> IV. VISITE INITIALE ---> V.4 Traitements ---> Date de début de traitement ARV.

FICHE DU PATIENT AVEC VIH - ENFANT ---> III. ANTECEDENTS DE L'ENFANT ---> PTME ---> ARV CHEZ L'ENFANT ----> Date de début de traitement ARV

IipperMangesh committed:
COMRS-191  import necessary concepts so that Indicator 2 report can be done

tmvumbi2 commented on configuration/concepts/concepts_for_form_adult_patient_with_hiv.csv:
```diff
@@ -0,0 +1,5 @@
+﻿Uuid,Members,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Data class,Data type
+e3f9c7ee-aa3e-4224-9d18-42e09b095ac6,,Date de début de traitement ARV,Date de début de traitement ARV,Date de début de traitement ARV,Date de début de traitement ARV,Misc,Date
```
@IipperMangesh Could you please use the following translations for french and english? 
Fully specified name:en -> ARV treatment start date
Fully specified name:fr -> Date de début de traitement ARV
Short specified name:en -> ARV treatment start date
Short specified name:fr -> Date de début de traitement ARV

tmvumbi2 commented on configuration/concepts/concepts_for_form_adult_patient_with_hiv.csv:
```diff
@@ -0,0 +1,5 @@
+﻿Uuid,Members,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Data class,Data type
+e3f9c7ee-aa3e-4224-9d18-42e09b095ac6,,Date de début de traitement ARV,Date de début de traitement ARV,Date de début de traitement ARV,Date de début de traitement ARV,Misc,Date
+20b2eac4-ff3a-4569-b610-067c4cd8b027,e3f9c7ee-aa3e-4224-9d18-42e09b095ac6,V.4 Traitements,V.4 Traitements [fr],V.4 Traitements,V.4 Traitements [fr],Misc,N/A
```
@IipperMangesh Could you please use the following translations for french and english?
Fully specified name:en -> V.4 Treatments
Fully specified name:fr -> V.4 Traitements
Short specified name:en -> V.4 Treatments
Short specified name:fr -> V.4 Traitements

tmvumbi2 commented on configuration/concepts/concepts_for_form_adult_patient_with_hiv.csv:
```diff
@@ -0,0 +1,5 @@
+﻿Uuid,Members,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Data class,Data type
+e3f9c7ee-aa3e-4224-9d18-42e09b095ac6,,Date de début de traitement ARV,Date de début de traitement ARV,Date de début de traitement ARV,Date de début de traitement ARV,Misc,Date
+20b2eac4-ff3a-4569-b610-067c4cd8b027,e3f9c7ee-aa3e-4224-9d18-42e09b095ac6,V.4 Traitements,V.4 Traitements [fr],V.4 Traitements,V.4 Traitements [fr],Misc,N/A
+059b8309-4cba-46cc-a857-e1e74be3fa86,20b2eac4-ff3a-4569-b610-067c4cd8b027,IV. VISITE INITIALE,IV. VISITE INITIALE [fr],IV. VISITE INITIALE,IV. VISITE INITIALE [fr],Misc,N/A
```
@IipperMangesh Could you please use the following translations for french and english?
Fully specified name:en -> IV. INITIAL VISIT
Fully specified name:fr -> IV. VISITE INITIALE
Short specified name:en -> IV. INITIAL VISIT
Short specified name:fr -> IV. VISITE INITIALE

tmvumbi2 commented on configuration/concepts/concepts_for_form_adult_patient_with_hiv.csv:
```diff
@@ -0,0 +1,5 @@
+e3f9c7ee-aa3e-4224-9d18-42e09b095ac6,,Date de début de traitement ARV,Date de début de traitement ARV,Date de début de traitement ARV,Date de début de traitement ARV,Misc,Date
+20b2eac4-ff3a-4569-b610-067c4cd8b027,e3f9c7ee-aa3e-4224-9d18-42e09b095ac6,V.4 Traitements,V.4 Traitements [fr],V.4 Traitements,V.4 Traitements [fr],Misc,N/A
+059b8309-4cba-46cc-a857-e1e74be3fa86,20b2eac4-ff3a-4569-b610-067c4cd8b027,IV. VISITE INITIALE,IV. VISITE INITIALE [fr],IV. VISITE INITIALE,IV. VISITE INITIALE [fr],Misc,N/A
+1fb2dd86-53b5-4815-9c64-edc081b908d9,059b8309-4cba-46cc-a857-e1e74be3fa86,FICHE DU PATIENT AVEC VIH - ADULTE,FICHE DU PATIENT AVEC VIH - ADULTE,FICHE DU PATIENT AVEC VIH - ADULTE,FICHE DU PATIENT AVEC VIH - ADULTE,Misc,N/A
```
@IipperMangesh Could you please use the following translations for french and english?
Fully specified name:en -> ADULT HIV PATIENT FORM
Fully specified name:fr -> FICHE DU PATIENT AVEC VIH - ADULTE
Short specified name:en -> ADULT HIV PATIENT FORM
Short specified name:fr -> FICHE DU PATIENT AVEC VIH - ADULTE

tmvumbi2 commented on configuration/concepts/concepts_for_form_enfant_patient_with_hiv.csv:
```diff
@@ -0,0 +1,6 @@
+﻿Uuid,Members,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Data class,Data type
+e3f9c7ee-aa3e-4224-9d18-42e09b095ac6,,Date de début de traitement ARV,Date de début de traitement ARV,Date de début de traitement ARV,Date de début de traitement ARV,Misc,Date
```
@IipperMangesh Could you please use the following translations for french and english?
Fully specified name:en -> ARV treatment start date
Fully specified name:fr -> Date de début de traitement ARV
Short specified name:en -> ARV treatment start date
Short specified name:fr -> Date de début de traitement ARV

tmvumbi2 commented on configuration/concepts/concepts_for_form_enfant_patient_with_hiv.csv:
```diff
@@ -0,0 +1,6 @@
+﻿Uuid,Members,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Data class,Data type
+e3f9c7ee-aa3e-4224-9d18-42e09b095ac6,,Date de début de traitement ARV,Date de début de traitement ARV,Date de début de traitement ARV,Date de début de traitement ARV,Misc,Date
+29955cb9-1fd8-442e-bc59-2f3b20e70617,e3f9c7ee-aa3e-4224-9d18-42e09b095ac6,ARV CHEZ L'ENFANT,ARV CHEZ L'ENFANT [fr],ARV CHEZ L'ENFANT,ARV CHEZ L'ENFANT [fr],Misc,N/A
```
@IipperMangesh Could you please use the following translations for french and english?
Fully specified name:en -> INFANT ARV
Fully specified name:fr -> ARV CHEZ L'ENFANT
Short specified name:en -> INFANT ARV
Short specified name:fr -> ARV CHEZ L'ENFANT

tmvumbi2 commented on configuration/concepts/concepts_for_form_enfant_patient_with_hiv.csv:
```diff
@@ -0,0 +1,6 @@
+e3f9c7ee-aa3e-4224-9d18-42e09b095ac6,,Date de début de traitement ARV,Date de début de traitement ARV,Date de début de traitement ARV,Date de début de traitement ARV,Misc,Date
+29955cb9-1fd8-442e-bc59-2f3b20e70617,e3f9c7ee-aa3e-4224-9d18-42e09b095ac6,ARV CHEZ L'ENFANT,ARV CHEZ L'ENFANT [fr],ARV CHEZ L'ENFANT,ARV CHEZ L'ENFANT [fr],Misc,N/A
+51d7342d-d8ad-4061-97f3-c32724a47694,29955cb9-1fd8-442e-bc59-2f3b20e70617,PTME,PTME [fr],PTME,PTME [fr],Misc,N/A
+300fffce-30ef-4c3e-a1be-f6acbe99c243,51d7342d-d8ad-4061-97f3-c32724a47694,III. ANTECEDENTS DE L'ENFANT,III. ANTECEDENTS DE L'ENFANT [fr],III. ANTECEDENTS DE L'ENFANT,III. ANTECEDENTS DE L'ENFANT [fr],Misc,N/A
```
@IipperMangesh Could you please use the following translations for french and english?
Fully specified name:en -> III. CHILD HISTORY
Fully specified name:fr -> III. ANTECEDENTS DE L'ENFANT
Short specified name:en -> III. CHILD HISTORY
Short specified name:fr -> III. ANTECEDENTS DE L'ENFANT

tmvumbi2 commented on configuration/concepts/concepts_for_form_enfant_patient_with_hiv.csv:
```diff
@@ -0,0 +1,6 @@
+29955cb9-1fd8-442e-bc59-2f3b20e70617,e3f9c7ee-aa3e-4224-9d18-42e09b095ac6,ARV CHEZ L'ENFANT,ARV CHEZ L'ENFANT [fr],ARV CHEZ L'ENFANT,ARV CHEZ L'ENFANT [fr],Misc,N/A
+51d7342d-d8ad-4061-97f3-c32724a47694,29955cb9-1fd8-442e-bc59-2f3b20e70617,PTME,PTME [fr],PTME,PTME [fr],Misc,N/A
+300fffce-30ef-4c3e-a1be-f6acbe99c243,51d7342d-d8ad-4061-97f3-c32724a47694,III. ANTECEDENTS DE L'ENFANT,III. ANTECEDENTS DE L'ENFANT [fr],III. ANTECEDENTS DE L'ENFANT,III. ANTECEDENTS DE L'ENFANT [fr],Misc,N/A
+48a724c1-fd24-45da-855c-33fcb4ce9c5d,300fffce-30ef-4c3e-a1be-f6acbe99c243,FICHE DU PATIENT AVEC VIH - ENFANT,FICHE DU PATIENT AVEC VIH - ENFANT,FICHE DU PATIENT AVEC VIH - ENFANT,FICHE DU PATIENT AVEC VIH - ENFANT,Misc,N/A
```
@IipperMangesh Could you please use the following translations for french and english?
Fully specified name:en -> CHILD HIV PATIENT FORM
Fully specified name:fr -> FICHE DU PATIENT AVEC VIH - ENFANT
Short specified name:en -> CHILD HIV PATIENT FORM
Short specified name:fr -> FICHE DU PATIENT AVEC VIH - ENFANT

tmvumbi2 commented on configuration/concepts/concepts_for_form_enfant_patient_with_hiv.csv:
```diff
@@ -0,0 +1,6 @@
+﻿Uuid,Members,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Data class,Data type
+e3f9c7ee-aa3e-4224-9d18-42e09b095ac6,,Date de début de traitement ARV,Date de début de traitement ARV,Date de début de traitement ARV,Date de début de traitement ARV,Misc,Date
+29955cb9-1fd8-442e-bc59-2f3b20e70617,e3f9c7ee-aa3e-4224-9d18-42e09b095ac6,ARV CHEZ L'ENFANT,ARV CHEZ L'ENFANT [fr],ARV CHEZ L'ENFANT,ARV CHEZ L'ENFANT [fr],Misc,N/A
+51d7342d-d8ad-4061-97f3-c32724a47694,29955cb9-1fd8-442e-bc59-2f3b20e70617,PTME,PTME [fr],PTME,PTME [fr],Misc,N/A
```
@IipperMangesh Could you please use the following translations for french and english? Note: I am assuming here that PTME is a french abbreviation (is this correct @rubailly ?) 
Fully specified name:en -> PTME [en]
Fully specified name:fr -> PTME
Short specified name:en -> PTME [en]
Short specified name:fr -> PTME

tmvumbi2 reviewed (changes requested):
Thanks for the PR @IipperMangesh , I suggested some french and english translations in the comments.

Please also note the import is failing, the following error is returned on the logs (I'll send to your slack the entire error stack).
`ERROR - CsvParser.saveAll(185) |2019-03-22 07:49:23,622| An OpenMRS object could not be constructed or saved from the following CSV line: [300fffce-30ef-4c3e-a1be-f6acbe99c243, 51d7342d-d8ad-4061-97f3-c32724a47694, III. ANTECEDENTS DE L'ENFANT, III. ANTECEDENTS DE L'ENFANT [fr], III. ANTECEDENTS DE L'ENFANT, III. ANTECEDENTS DE L'ENFANT [fr], Misc, N/A]
java.lang.IllegalArgumentException: 'uuid' was not found in the index map: {data class=6, members=1, ?uuid=0, short name:fr=5, fully specified name:en=2, short name:en=4, fully specified name:fr=3, data type=7}
`

Finally, please also add the 2 new forms to the concept set `All Observation Templates` so that they can be accessed from the consultation module.

IipperMangesh committed:
COMRS-191 update both Adult and Child form hierarchy as per requirement

tmvumbi2 committed:
COMRS-191 Update 'All Observation Templates' set

This commit updates the set 'All Observation Templates':
- to include the 2 new forms: adult and child hiv patient
- to remove the forms not used for currently

This commit also fixes charset issues on french words with accents; and uses a single CSV file for the 2 forms (so that the update of the set can be done when all the concepts for the 2 forms have been imported);
